sound/soc/soc-topology.c

```diff
@@ -2577,6 +2596,13 @@ int snd_soc_tplg_component_remove(struct snd_soc_component *comp, u32 index)
 			case SND_SOC_DOBJ_DAI_LINK:
 				remove_link(comp, dobj, pass);
 				break;
+			case SND_SOC_DOBJ_BE_DAI_LINK:
+				/*
+				 * we don't free the BE link since it is not
+				 * allocated by topology
```

Comment:
You should describe what the unload_link does, since the link_load() only deals with initializations according to the comments. is this to free private data handled by the component driver and allocated by the link_load

Reply:
@plbossart It is designed for deinitialization. I think it can also be used to free private data allocated by the link_load.

Reply:
agree, I just want this to be mentioned in the comments. no functional change